### PR TITLE
Modern theme for vertical tabs

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**
https://blackbaud.invisionapp.com/console/share/UG1N3XTCC7/496056519

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-tabs?_sp.spa=pr-vertical-tab-modern-theme](https://developer.blackbaud.com/skyux-tabs?_sp.spa=pr-vertical-tab-modern-theme)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-tabs?_sp.spa=pr-vertical-tab-modern-theme](https://developer.blackbaud.com/skyux-tabs?_sp.spa=pr-vertical-tab-modern-theme)

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-tabs?_sp.spa=pr-vertical-tab-modern-theme](https://developer.blackbaud.com/skyux-tabs?_sp.spa=pr-vertical-tab-modern-theme)

File: src/app/public/modules/vertical-tabset/vertical-tabset-group.component.html [hunk 1]
```diff
@@ -11,14 +11,34 @@
...
+    <sky-chevron *ngIf="themeName === 'default'"
       class="sky-vertical-tabset-group-header-chevron"
       [direction]="open ? 'up' : 'down'"
       [disabled]="disabled"
       (directionChange)="toggleMenuOpen()"
       (keyup.enter)="$event.stopPropagation()"
     >
     </sky-chevron>
+    <button *ngIf="themeName === 'modern' && !open"
+      class="sky-btn sky-vertical-tabset-group-header-chevron"
```
Comment (Blackbaud-ToddRoberts) [inline src/app/public/modules/vertical-tabset/vertical-tabset-group.component.html:23]: i think we should be including the sky-btn-icon-borderless class here, may need to adjust the alignment of it.

**Review by Blackbaud-ToddRoberts — CHANGES_REQUESTED** [review]

File: src/app/public/modules/vertical-tabset/vertical-tabset-group.component.html (hunk 1 above)
Comment (Blackbaud-AlexKingman) [reply src/app/public/modules/vertical-tabset/vertical-tabset-group.component.html]: Good call. Done.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-tabs?_sp.spa=pr-vertical-tab-modern-theme](https://developer.blackbaud.com/skyux-tabs?_sp.spa=pr-vertical-tab-modern-theme)

**Review by Blackbaud-ToddRoberts — APPROVED** [review]

**Pull request merged**